spec/models/project_mediaflux_spec.rb

```diff
@@ -59,14 +59,6 @@
         end
         let(:project_parent) { Rails.configuration.mediaflux["api_root_collection"] }
 
-        before do
-          # Destroy the parent collection
-          Mediaflux::Http::NamespaceDestroyRequest.new(
-            session_token: current_user.mediaflux_session,
-            namespace: project_parent
-          ).destroy
-        end
-
```

Comment:
Why did you remove this? This test section is "when the parent collection does not exist", so ensuring that the parent collection does not exist before the tests run is important.

Reply:
I deleted it because I was under the impression that whenever the test runs, it automatically deletes everything in mediaflux by default since it has the `connect_to_mediaflux:` tag at the describe level so I assumed it was redundant and attempting to delete the namespace twice. Maybe i'm wrong about that?